### DISCUSSION TITLE
Integration with gerrit

### DISCUSSION THREAD

**Discussion by AriJang** [community/community#59111] [New to GitHub]
Github Community,

I am currently in the process of integrating Gerrit with GitHub, following the guide provided here: 
[link to the guide] (https://gerrit.googlesource.com/plugins/github).

Upon completion of the setup process, I was able to view and connect to public repositories in my Gerrit server. However, I'm facing an issue where my private repositories on GitHub do not appear on the list.

Furthermore, while there's a link provided to "Login with a different GitHub Scope", clicking on it does not show any list or options in the "Login Scope Selection" screen.

Currently, I am using the free plan of GitHub. I'm wondering if this issue is related to my plan and whether private repository integration is only available with paid GitHub plans. 
Or should I set more configuration ?

I would appreciate any insights or suggestions on how to resolve this issue. Thank you for your help in advance!

**Comment by queenofcorgis** [comment]
Hi there @AriJang, 

Thanks for writing in! Have you spoken to Gerrit at all regarding why private repositories won't appear in the list?

**Comment by AriJang** [comment]
no, yet.
Do you think this should be checked by Gerrit side? 
Below is my issue screenshot. 
There is no private repository in the list even if I have several private repositories. 
Even more, I tried to change my scope but that also does not have anything to me. 
<img width="633" alt="screenshot" src="https://github.com/community/community/assets/128448785/eafc11af-f0b2-44c9-b158-e80e3838108a">
